src/App.tsx

```diff
@@ -1,61 +1,126 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { useState } from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { User } from './components/types/User';
+import { Todo } from './components/types/Todo';
+
+function getUser(userId: number): User | null {
```

Comment:
don't forget about past checklists
```suggestion
function getUserById(id: number): User | null {
```